queries.sql

```diff
@@ -37,13 +37,8 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT name FROM animals 
 WHERE escape_attempts = (SELECT MAX(escape_attempts) FROM animals);
 
```

Comment:
- [ ] This query returns the wrong result. The query states you should find which category of animals (`neutered` or `not neutered`) that escapes the most. In other words, you should select the `neutered`, then find out the total `escape attempts` of those who are neutered and those who are not neutered. You can then return the one with the highest. Kindly update your screenshot when you get the correct result. 💻 

Hint: You would want to group by neutered column. 👍